### PR TITLE
[Claudia Movradin] - New branch

### PR DESCRIPTION
**Pull request by Claudia-20**

### CODE REVIEW

File: Claudia/src/main/java/FirstTask/Month.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package FirstTask;
+
+public class Month {
+    public static void main(String[] args) {
+        int month = 9;
```
Comment (venache1) [inline Claudia/src/main/java/FirstTask/Month.java]: What will be the output for month = 45?

File: Claudia/src/main/java/FirstTask/Month.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            case 10:
+                System.out.println("October");
+                break;
+            case 11:
+                System.out.println("November");
+                break;
+            case 12:
+                System.out.println("December");
+        }
+    }
```
Comment (venache1) [inline Claudia/src/main/java/FirstTask/Month.java]: It's a good approach to extract this logic in a separate method for reusability. For example if I want to print 3rd, 6th and 10th month at the same time.

File: Claudia/src/main/java/FirstTask/PrimitiveData.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package FirstTask;
+
+public class PrimitiveData {
+    public static void main(String[] args) {
+
+        double radius;
+        double area;
+        final double pi = 3.145658;
```
Comment (venache1) [inline Claudia/src/main/java/FirstTask/PrimitiveData.java]: Could you use **Math.PI** constant?

File: Claudia/src/main/java/FirstTask/Month.java (hunk 1 above)
Comment (Claudia-20) [reply Claudia/src/main/java/FirstTask/Month.java]: Nothing will be displayed for month = 45. I will fix the code. Thanks!

File: Claudia/src/main/java/FirstTask/PrimitiveData.java (hunk 3 above)
Comment (Claudia-20) [reply Claudia/src/main/java/FirstTask/PrimitiveData.java]: Yes.

File: Claudia/src/main/java/FirstTask/Month.java (hunk 1 above)
Comment (Claudia-20) [reply Claudia/src/main/java/FirstTask/Month.java]: Done

File: Claudia/src/main/java/FirstTask/Month.java (hunk 2 above)
Comment (Claudia-20) [reply Claudia/src/main/java/FirstTask/Month.java]: Done

File: Claudia/src/main/java/FirstTask/PrimitiveData.java (hunk 3 above)
Comment (Claudia-20) [reply Claudia/src/main/java/FirstTask/PrimitiveData.java]: Done

File: Claudia/src/main/java/FirstTask/Numbers.java [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package FirstTask;
+
+public class Numbers {
+    public static void main(String[] args) {
+        System.out.println("Result is : ");
+
```
Comment (venache1) [inline Claudia/src/main/java/FirstTask/Numbers.java]: Remove empty line

File: Claudia/src/main/java/FirstTask/PrimitiveData.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
+package FirstTask;
+
+public class PrimitiveData {
+    public static void main(String[] args) {
+
```
Comment (venache1) [inline Claudia/src/main/java/FirstTask/PrimitiveData.java]: Remove empty line

File: Claudia/src/main/java/FirstTask/PrimitiveData.java [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+        double radius;
+        double area;
+
```
Comment (venache1) [inline Claudia/src/main/java/FirstTask/PrimitiveData.java]: Remove empty line

File: Claudia/src/main/java/FirstTask/PrimitiveData.java [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+        radius = 5;
```
Comment (venache1) [inline Claudia/src/main/java/FirstTask/PrimitiveData.java]: Why you didn't initialize this variable on line 6?

File: Claudia/src/main/java/FirstTask/String2.java [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+        if (text.equalsIgnoreCase(text1))
+            System.out.println("The strings are equal ");
+        else
+            System.out.println("The strings are Not equal");
+
+
+        String text2 = text.concat(text1);
+        System.out.println("The concatenated string: " + text2);
+    }
```
Comment (venache1) [inline Claudia/src/main/java/FirstTask/String2.java]: Please remove all empty lines

File: Claudia/src/main/java/FirstTask/PrimitiveData.java [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+        area = Math.PI * Math.pow(radius, 2);
+        System.out.println("Area of the circle with radius " + radius + " is " + area);
+
+    }
+
+}
```
Comment (venache1) [inline Claudia/src/main/java/FirstTask/PrimitiveData.java]: Remove all empty lines

**Comment by venache1** [comment]
Please squash your commits after fixing all comments!

File: Claudia/src/main/java/FirstTask/Numbers.java (hunk 4 above)
Comment (Claudia-20) [reply Claudia/src/main/java/FirstTask/Numbers.java:6]: Done

Comment (Claudia-20) [reply Claudia/src/main/java/FirstTask/Numbers.java]: Done

File: Claudia/src/main/java/FirstTask/PrimitiveData.java (hunk 5 above)
Comment (Claudia-20) [reply Claudia/src/main/java/FirstTask/PrimitiveData.java]: Done

File: Claudia/src/main/java/FirstTask/PrimitiveData.java (hunk 6 above)
Comment (Claudia-20) [reply Claudia/src/main/java/FirstTask/PrimitiveData.java]: Done

File: Claudia/src/main/java/FirstTask/PrimitiveData.java (hunk 7 above)
Comment (Claudia-20) [reply Claudia/src/main/java/FirstTask/PrimitiveData.java]: Done

File: Claudia/src/main/java/FirstTask/String2.java (hunk 8 above)
Comment (Claudia-20) [reply Claudia/src/main/java/FirstTask/String2.java]: Done

File: Claudia/src/main/java/FirstTask/PrimitiveData.java (hunk 9 above)
Comment (Claudia-20) [reply Claudia/src/main/java/FirstTask/PrimitiveData.java]: Done

**Review by venache1 — APPROVED** [review]

**Review by irinazota — APPROVED** [review]

**Review by VitalinaPetrenco — APPROVED** [review]

**Pull request merged**